cmd/daemonSetCheck/Dockerfile

```diff
@@ -1,9 +1,12 @@
-FROM golang:1.13 AS builder
+FROM golang:1.13-alpine AS builder
+RUN apk upgrade
```

Comment:
Is `RUN apk upgrade` necessary here? You're upgrading alpine packages in the builder and then throwing it away. The go binary is all that will be running in this container.

Reply:
it could matter because it will cause the binary to be produced with the newest versions of build software and libraries.  I would leave it in, I think.